docs/How/release_process.md

```diff
@@ -1,2 +1,88 @@
-# Overview
-
+## Overview
+
+!!! warning "Contents and Process Under Active Development"
+    The basics of this process are identical to those followed by the Project Mu firmware integration and release process internal to Microsoft, but the formal documentation, branch naming, and tagging process is a work in progress. While this is how we expect things to work, there may be changes within the the first few releases driven by feedback within the team and any external consumers/contributors.
+
+In the interest of maintaining a close, well-defined relationship with the upstream project, TianoCore, the active release branch of Project Mu is periodically deprecated and all Mu-related changes are rebased onto a selected commit of TianoCore. This keeps Project Mu up to date with TianoCore while highlighting all Project Mu differences in the most recent commits and encouraging the reverse integration of all universal changes/fixes back into TianoCore.
+
+In general, the life-cycle of active code follows the following path:
+
+<center>![Stable Release, Upstream Rebase, Build/Boot Fixes, Active Development, Stabilization, Stable Release](../img/release_cycle.png)</center>
+
+All active work in Project Mu is performed on a `release/*` branch, usually named sequentially according to the date of TianoCore commit that it's based on (eg. `release/201808` is based on the `edk2-stable201808` branch in TianoCore). Work proceeds on that branch until a new TianoCore integration is targeted, at which point a new branch in created and all existing changes are rebased onto the new branch and the new branch is used for all active development going forward. At this point, the previous branch enters a stabilization period where further tests are performed and only bug fixes are allowed to be committed. After stabilization, the branch is labeled as `stable` and will only receive critical bug fixes either directly to the branch or backported from a more recent release.

```

Comment:
How about we remove "usually".  If we later find an exception, we can document that.
Also, "eg." should have 2 periods "e.g."